cpp/include/ray/api.h

```diff
@@ -162,6 +162,11 @@ PlacementGroup GetPlacementGroup(const std::string &name);
 /// Returns true if the current actor was restarted, otherwise false.
 bool WasCurrentActorRestarted();
 
+/// Get the namespace of this job.
+inline std::string GetNamespace() {
+  return ray::internal::GetRayRuntime()->GetNamespace();
```

Comment:
Please keep API declaration here and move the implementation to `inline implementation` part below. We should make a clear API list to users.

Reply:
ok